web/api/v1/node.ts

```diff
@@ -1,26 +1,66 @@
 import { V1Node, V1NodeList } from "@kubernetes/client-node";
-import { instance } from "@/api/v1/index";
+import { k8sInstance } from "@/api/v1/index";
+import axios from "axios";
 
 export const applyNode = async (req: V1Node, onError: (_: string) => void) => {
   try {
-    const res = await instance.post<V1Node>(`/nodes`, req);
+    if (!req.metadata?.name) {
+      onError("metadata.name is not provided");
+      return;
+    }
+    const res = await k8sInstance.patch<V1Node>(
+      `/nodes/${req.metadata.name}?fieldManager=simulator`,
+      req,
+      { headers: { "Content-Type": "application/strategic-merge-patch+json" } }
+    );
     return res.data;
   } catch (e: any) {
-    onError(e);
+    if (axios.isAxiosError(e) && e.response && e.response.status === 404) {
+      const res = await createNode(req, onError);
+      return res;
+    }
+    onError("Caused by applyNode: " + e);
```

Comment:
Please change other wordings as well.

```suggestion
    onError("failed to apply Node: " + e);
```

Reply:
fixed: 3f20091